### DISCUSSION TITLE
Force enable persistent maps

### DISCUSSION THREAD

**Discussion by matte-oss** [rand256/valetudo#525] [Q&A] [answered]
I have an old Xiaomi vacuum v1 that deletes  the map every time you start a cleaning.

I installed Valetudo RE on It, Is there a way to force the persistent Maps?
Thanks in advance

**Comment by pidator** [comment, marked as answer]
On Gen1 the only way to have a "kind of" persistent maps is to only use the zone cleaning function. This means: let the robot create the whole map in the first run and then create zones to clean. With RE you can save (and restore) the (complete) map too which is very helpful if there's an error during the run and the map got deleted!
